fluent-syntax/src/errors.js

```diff
@@ -1,26 +1,7 @@
 export function error(ps, message) {
   const err = new SyntaxError(message);
```

Comment:
Are `SyntaxErrors` OK for now, or would it be better to start with our own `Annotation` class here?